### ISSUE TITLE
Can we give multiple images as input in yolov2/yolov3 framework?

### ISSUE THREAD

**Issue by murari023** [AlexeyAB/darknet#1136]
I was wondering if i can give multiple images as input (suppose 10 images). So the input layer will have size of height*width*depth*no._of_images, like we do in keras.

**Comment by AlexeyAB** [comment]
There is no try batch processing, but you can pass multiple images to the Darknet - read more: https://github.com/AlexeyAB/darknet/issues/1099#issuecomment-400501945

**Comment by murari023** [comment]
Thanx for the answer. But this code for getting detection results on multiple images. Giving them input as a batch. 
I am trying to give 10 continuous frames of a video as input to the YOLO network and get detection result for only one particular (last) frame. So the bboxes ground truth of the 10th frame would be used for loss calculations. Is there a way to do this?

**Comment by AlexeyAB** [comment]
No, there is not a way to do this by default.
You should change source code to this, in such places: https://github.com/AlexeyAB/darknet/issues/764#issuecomment-387124979
